netket/graph/grid.py

```diff
@@ -145,6 +145,80 @@ def translate(el, sites):
 
         return [translate(el, identity) for el in translation_group]
 
+    def space_group(self, identity=None) -> List[List[int]]:
```

Comment:
`identity` is probably no longer the best term here.

If you pass `identity=[[[0, 1, 2, ..., N - 1]]]`, that is, the actual identity permutation, you would get only the space group, right?

I'm not sure what a better name is at the moment, though.

Reply:
Yes, the space group is rotations and reflections. I think a rotation is the coordinate mapping (i,j....)->(-j,i...) which I've accomplished with two reflections. I can do this, just have to think a bit about how do handle ndim > 2.

I will think about a new name for "identity".